src/components/stats/SoloCommits.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react';
+
+import { Query } from 'react-apollo';
+import { gql } from 'apollo-boost';
+import client from '../../utils/GitHubGQL';
+
+class SoloCommits extends Component {
+  constructor(props) {
+    super();
+    this.state = {
+      startDate: new Date(props.startDate).toISOString(),
+      endDate: `${props.endDate}T23:59:59.999Z`,
+      username: props.username,
+      repository: props.repository,
+      winner: '',
+      highScore: 0,
+    };
+  }
+
+  componentDidMount() {
+    const { username } = this.state;
+    const { repository } = this.state;
+    const { startDate } = this.state;
+    const { endDate } = this.state;
+
+    client.query({
+      query: gql`
+      {
+        repository(owner: "${username}", name: "${repository}") {
+          defaultBranchRef {
+            target {
+              ... on Commit {
+                history(since: "${startDate}" until: "${endDate}") {
+                  nodes {
+                    additions
+                    author {
+                      user {
+                        login
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }`,
+    })
```

Comment:
Small one but I’d prob just run your autoformatter on this file

Reply:
auto formatter wanted this... :)?